addons/mail/static/tests/composer/composer.test.js

```diff
@@ -1054,3 +1054,76 @@ test("Canned response last used changes on posting", async () => {
     cannedResponse = pyEnv["mail.canned.response"].search_read([["id", "=", cannedResponseId]])[0];
     expect(cannedResponse.last_used).not.toBeEmpty();
 });
+
+test("Canned response suggestions are not focused by default", async () => {
```

Comment:
Test is easier to read in active form: `Does not auto-select 1st canned response suggestion`